lib/metrics/registry.go

```diff
@@ -61,19 +61,39 @@ func (r *Registry) NewMetric(name string, typ stats.MetricType, t ...stats.Value
 	oldMetric, ok := r.metrics[name]
 
 	if !ok {
-		m := stats.New(name, typ, t...)
+		valueType := stats.Default
```

Comment:
It seems like a small method (like it was before). Do you think we can benefit from moving this into the other constructor? :thinking:

Reply:
I ended up renaming `stats.New` to `stats.newMetric`, keeping it private, to make things simpler 👍🏻